src/components/touchables/GenericTouchable.tsx

```diff
@@ -250,20 +255,22 @@ export default class GenericTouchable extends Component {
       accessible: this.props.accessible !== false,
       accessibilityLabel: this.props.accessibilityLabel,
       accessibilityHint: this.props.accessibilityHint,
-      accessibilityComponentType: this.props.accessibilityComponentType,
       accessibilityRole: this.props.accessibilityRole,
-      accessibilityStates: this.props.accessibilityStates,
-      accessibilityTraits: this.props.accessibilityTraits,
+      // TODO: check if changed to no 's' correctly, also removed 2 props that are no longer available: `accessibilityComponentType` and `accessibilityTraits`,
+      // would be good to check if it is ok for sure, see: https://github.com/facebook/react-native/issues/24016
+      accessibilityState: this.props.accessibilityState,
       nativeID: this.props.nativeID,
       onLayout: this.props.onLayout,
       hitSlop: this.props.hitSlop,
+      interpolate: undefined,
     };
 
     return (
       <BaseButton
         style={this.props.containerStyle}
         onHandlerStateChange={
-          this.props.disabled ? null : this.onHandlerStateChange
+          // TODO: not sure if it can be undefined instead of null
+          this.props.disabled ? undefined : this.onHandlerStateChange
```

Comment:
Why this change was introduced?

Reply:
Because `onHandlerStateChange` does not accept null. I think `undefined` is safe since there is a check in code of `BaseButton` (`this.props.onHandlerStateChange?.(e)`) that will treat `null` and `undefined` the same.

Reply:
Unresolving for visibility.